### DISCUSSION TITLE
Dynamic Power Limiter Einstellungen, PV+Batterie

### DISCUSSION THREAD

**Discussion by ar12321** [hoylabs/OpenDTU-OnBattery#1302] [General]
Ich betreibe am HM600 ein PV-Paneel und am zweiten Eingang eine Batterie (über Victron Laderegler).
Meine Frage, wie erkennt openDTU bzw. der Dynamic Power Limiter ob an 1 oder 2 HM600-Eingängen eine Leistung bereitgestellt wird? (bei nur einem Eingang muss die Regelung den doppelten Leistungswert für einen Eingang beachten),
Wäre es möglich über eine Variable die Regelung zu beeinflussen (z.B. Grenzwert für PV-Leistung am 1. Eingang < 10W).
Weiterhin wäre es praktisch, wenn ohne Batterie-Lieferung (Spannung am Eingang 2 < 10V) der Power-Limiter den HM600 auf maximaler Leistung belässt.
Vielleicht gibt es schon diese Einstellungen, ich finde sie nur nicht.

**Comment by schlimmchen** [comment]
> Ich betreibe am HM600 ein PV-Paneel und am zweiten Eingang eine Batterie (über Victron Laderegler).

Wird nicht unterstützt. Siehe auch [die erste FAQ der Doku](https://opendtu-onbattery.net/firmware/faq/).

**Comment by ar12321** [comment]
Hallo, danke für die schnelle Antwort.

Trotzdem erkennt DPL ob ein oder zwei Eingänge am HM600 Spannung/ Leistung haben. Die Leistungsanforderung berücksichtigt bei zwei Eingängen den "einfachen" Leistungsanforderungswert, bei einem aktiven Eingang den doppelten Wert. Über welche Grenzwerte (Spannung, PV-Leistung ?) wird dies erkannt?

**Comment by schlimmchen** [comment]
Das macht nicht der DPL, sondern der Inverter. Da haben wir uns schon oft drüber unterhalten, aber das ist immer recht schwer zu finden. Jedenfalls ist es so, dass der Inverter das gesetzte Limit gleichmäßig auf alle MPPTs verteilt. Also auch nicht gleichmäßig auf alle Eingäge -- je nach Invertermodell.

Hast du dir mal die Funktion "Überskalierung erlauben um Verschattung auszugleichen"? Dabei wird das Limit beispielsweise verdoppelt, wenn klar zu sein scheint, das ein Eingang keine Leistung liefert (wie in deinem Fall: Batterie abgeklemmt). Vielleicht hilft dir das weiter?

**Comment by ar12321** [comment]
Ich finde leider nicht für Funktion "Überskalierung".
Dafür habe ich gefunden Eingang für Spannungsmessung. Was bewirkt dieses?

**Comment by schlimmchen** [comment]
https://opendtu-onbattery.net/firmware/configuration/dpl/#compensate-for-shading Ich hab vergessen, dass die Funktion nur greift, wenn der Inverter als solar-betrieben eingestellt ist.

> Dafür habe ich gefunden Eingang für Spannungsmessung. Was bewirkt dieses?

https://opendtu-onbattery.net/firmware/configuration/dpl/#input-used-for-voltage-measurement

**Comment by ar12321** [comment]
Danke für die Antworten. 
Offensichtlich bekommt man mit den HM600 meine Vorstellungen nicht realisiert, da nur eine gemeinsame Regelung beider Eingänge möglich ist.

**Reply by spcqike** [reply]
Das ist bei allen Hoymiles so. Daher wird ein Mischbetrieb, wie du es beschreibst, auch nicht unterstützt.

**Comment by schlimmchen** [comment]
> da nur eine gemeinsame Regelung beider Eingänge möglich ist.

So ist es.
